DigitalLearningSolutions.Data/Services/LoginService.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace DigitalLearningSolutions.Data.Services
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Models.User;
+
+    public interface ILoginService
+    {
+        public (AdminUser?, List<DelegateUser>) GetUsersByUsername(string username);
```

Comment:
I think this should probably be in the UserService instead since it doesn't strictly deal with Login/Out (and would potentially be useful on the MyAccount page which shouldn't be using a LoginService for example)

Similarly GetVerifiedAdminUserAssociatedWithApprovedDelegateUser?

Reply:
I'd argue that GetVerifiedAdminUserAssociatedWithApprovedDelegateUser should probably stay in LoginService as it also takes in an input password to verify against the account. But GetUsersByUsername can most certainly be put in the UserService.

Reply:
Yeah that makes sense, didn't notice it took a password